<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>levelingServices</class>
 <widget class="QWidget" name="levelingServices">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lytLevelSrvics">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>811</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lytLevelSrvics
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/lytLevelSrvics.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="stopJack1Btn">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>20</y>
     <width>73</width>
     <height>73</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#stopJack1Btn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/stopLvlBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="stopJack2Btn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>20</y>
     <width>73</width>
     <height>73</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#stopJack2Btn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/stopLvlBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="stopJack3Btn">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>367</y>
     <width>73</width>
     <height>73</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#stopJack3Btn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/stopLvlBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="stopAllJackBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>367</y>
     <width>73</width>
     <height>73</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#stopAllJackBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/stopLvlBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="stopJack4Btn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>367</y>
     <width>73</width>
     <height>73</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#stopJack4Btn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/stopLvlBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="dwnJack3Btn">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>365</y>
     <width>58</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#dwnJack3Btn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/dwnBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="dwnAllJackBtn">
   <property name="geometry">
    <rect>
     <x>293</x>
     <y>365</y>
     <width>58</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#dwnAllJackBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/dwnBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="dwnJack4Btn">
   <property name="geometry">
    <rect>
     <x>574</x>
     <y>365</y>
     <width>58</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#dwnJack4Btn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/dwnBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="dwnJack1Btn">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>20</y>
     <width>58</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#dwnJack1Btn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/dwnBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="dwnJack2Btn">
   <property name="geometry">
    <rect>
     <x>574</x>
     <y>20</y>
     <width>58</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#dwnJack2Btn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/dwnBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="upJack1Btn">
   <property name="geometry">
    <rect>
     <x>157</x>
     <y>41</y>
     <width>58</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#upJack1Btn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/upBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="upJack2Btn">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>41</y>
     <width>58</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#upJack2Btn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/upBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="upJack4Btn">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>387</y>
     <width>58</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#upJack4Btn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/upBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="upAllJackBtn">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>387</y>
     <width>58</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#upAllJackBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/upBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="upJack3Btn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>387</y>
     <width>58</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#upJack3Btn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/upBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pwrMotorBtn">
   <property name="geometry">
    <rect>
     <x>444</x>
     <y>66</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pwrMotorBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/offBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="highSpeedValBtn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>296</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#highSpeedValBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/offBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="lowSpeedValBtn">
   <property name="geometry">
    <rect>
     <x>504</x>
     <y>296</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lowSpeedValBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/offBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bckLvlingBtn">
   <property name="geometry">
    <rect>
     <x>634</x>
     <y>283</y>
     <width>156</width>
     <height>69</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#bckLvlingBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/bckLvlingBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="jck1PrxyBtn">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>137</y>
     <width>22</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#jck1PrxyBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="jck2PrxyBtn">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>166</y>
     <width>22</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#jck2PrxyBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="jck3PrxyBtn">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>194</y>
     <width>22</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#jck3PrxyBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="jck4PrxyBtn">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>221</y>
     <width>22</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#jck4PrxyBtn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="disVal">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>248</y>
     <width>22</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#disVal
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="prsSw1Btn">
   <property name="geometry">
    <rect>
     <x>543</x>
     <y>137</y>
     <width>22</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#prsSw1Btn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="prsSw2Btn">
   <property name="geometry">
    <rect>
     <x>543</x>
     <y>165</y>
     <width>22</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#prsSw2Btn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="prsSw3Btn">
   <property name="geometry">
    <rect>
     <x>543</x>
     <y>193</y>
     <width>22</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#prsSw3Btn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="prsSw4Btn">
   <property name="geometry">
    <rect>
     <x>543</x>
     <y>221</y>
     <width>22</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#prsSw4Btn
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="mnPrsSw">
   <property name="geometry">
    <rect>
     <x>543</x>
     <y>248</y>
     <width>22</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#mnPrsSw
{
background-color: trasparent;
background: none;
border: none;
background-repeat: none;
border-image: url(:/crclRdBtn.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="xLbl">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>140</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#xLbl
{
color: rgb(50, 50, 50) ;
}</string>
   </property>
   <property name="text">
    <string>X = </string>
   </property>
  </widget>
  <widget class="QLabel" name="xLblVal">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>140</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#xLblVal
{
color: rgb(50, 50, 50) ;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="yLbl">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>210</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#yLbl
{
color: rgb(50, 50, 50) ;
}</string>
   </property>
   <property name="text">
    <string>Y = </string>
   </property>
  </widget>
  <widget class="QLabel" name="yLblVal">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>210</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#yLblVal
{
color: rgb(50, 50, 50) ;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
